data/notebooks/Export_Table_ACLs.py

```diff
@@ -0,0 +1,177 @@
+# Databricks notebook source
+# MAGIC %md #Export Table ACLs
+# MAGIC 
+# MAGIC Exports Table ACLS to a file on DBFS, which can be imported using the Import_Table_ACLs script
+# MAGIC 
+# MAGIC Parameters:
+# MAGIC - Databases: [Optional] comma separated list of databases to be exported, if empty all databases will be exported
+# MAGIC - OutputPath: Path to write the exported file to (Choose a something under DBFS:/user/hive/warehouse, there an admin has write access)
+# MAGIC - OutputFormat: JSON
+# MAGIC 
+# MAGIC Execution: Run the notebook on a cluster with Table ACL's enabled as a user who is an admin 
+# MAGIC 
+# MAGIC Supportes ACLs for Object types:
+# MAGIC - Catalog: included if all databases are exported, not included if databases to be exported are specified
+# MAGIC   - Database: included
+# MAGIC     - Table: included
+# MAGIC     - View: included
+# MAGIC     -
+# MAGIC 
+# MAGIC WARNING: Needs more testing and fixing ... so far all I know with certainty is, that it does not work quite right yet
+# MAGIC 
+# MAGIC tomi.schumacher@databricks.com
+
+# COMMAND ----------
+
+#dbutils.widgets.removeAll()
+dbutils.widgets.text("Databases","tomi_schumacher_adl_test, tomi_schumacher_adl_test_restricted","1: Databases (opt)")
```

Comment:
Please remove any reference to your name in these parameters. Let's set some defaults here.

Reply:
done